src/OpenTelemetry/Trace/TracerProviderBuilderExtensions.cs

```diff
@@ -91,6 +91,21 @@ public static TracerProviderBuilder AddProcessor(this TracerProviderBuilder trac
             return tracerProviderBuilder;
         }
 
+        /// <summary>
+        /// Disable status switch for Activity.
+        /// </summary>
+        /// <param name="tracerProviderBuilder">TracerProviderBuilder instance.</param>
+        /// <returns>Returns <see cref="TracerProviderBuilder"/> for chaining.</returns>
+        public static TracerProviderBuilder DisableActivityStatusSwitch(this TracerProviderBuilder tracerProviderBuilder)
```

Comment:
Since the switch is static (global), we might want to just expose it statically. Kind of like we do for the global propagator. I think having these extensions on TracerProviderBuilder might mislead users into thinking the setting is per-tracer.